rel/server/env.sh.eex

```diff
@@ -4,5 +4,5 @@ export RELEASE_MODE=interactive
 cookie_path="${RELEASE_ROOT}/releases/COOKIE"
 if [ ! -f $cookie_path ]; then
   RELEASE_COOKIE=$(cat /dev/urandom | env LC_ALL=C tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
-  [[ $RELEASE_COOKIE > $cookie_path ]] || export RELEASE_COOKIE
+  [ $RELEASE_COOKIE > $cookie_path ] || export RELEASE_COOKIE
```

Comment:
Yup this code was broken but I just fixed it in main. Can you please revert this change and rebase? :)

Reply:
Reverted and rebased in latest commit.